Corona-Warn-App/src/main/assets/jsonschema-dcc-ebb6720.json

```diff
@@ -122,8 +123,8 @@
         "gnt": {
           "title": "Standardised forename",
           "description": "The forename(s) of the person, transliterated ICAO 9303",
-          "type": "string",
-          "pattern": "^[A-Z<]*$",
+          "__comment": "Special case handling: original pattern check dropped",
+          "__pattern": "^[A-Z<]*$",
```

Comment:
does this affect normal cases ?

Reply:
I don't think it should. I've copied this from the tech spec https://github.com/corona-warn-app/cwa-app-tech-spec/pull/85/files#diff-2870955822e4772ae4e76684d41910ca95b6462cfcc2029016509bcde80701bd. Since the pattern check is simply dropped it shouldn't have any effect on existing certs and from my testing, the certs I've previously had were read just fine.